src/hooks/useDownload.ts

```diff
@@ -77,9 +84,11 @@ async function internal<T>({ src, dst, headers, ephemeral, logger }: DownloadOpt
       dst.parent().mkpath()
       await body(reader, f, sz)
 
-      //TODO etags too
       const text = rsp.headers.get("Last-Modified")
+      const etag = rsp.headers.get("etag")
+
       if (text) mtime_entry().write({ text, force: true })
+      if (etag) etag_entry().write({text: etag, force: true})
```

Comment:
Supremely minor: I'd use consistent spacing on object notation on parallel lines.

I know our codebase as it stands would currently fail, but do you still like to "see how [you] feel" w.r.t. object notation spacing in TS, @mxcl, or should we adopt rigid guidelines? (I think we could also make this happen via eslint/prettier/etc rules and auto-application. This might prevent me from introducing semi-colons (though I don't believe I have done this in some time), and we might be able to catch imports which don't use the import map, though that might require some custom logic.)